test/factories/factories.rb

```diff
@@ -27,6 +27,11 @@
     username
     password "password"
     role 0
+    email_address
+    street_address "123 Maple Drive"
+    city "Denver"
+    state "CO"
+    zipcode 80231
```

Comment:
I don't think we need to do this with a zipcode, it doesn't make any sense and makes it more confusing to read.